panopticum/management/commands/importer.py

```diff
@@ -0,0 +1,1085 @@
+                                                          'virtual appliances packaged in OVF format that you can '
+                                                          'download and deploy in your vSphere environment.'},
+         'TBD': {'order': 12, 'description': 'To be filled later by user.'}}
+
+    depl_location_mapping = \
+        {'cloud-only': 'Cloud datacenter',
+         'cloud': 'Cloud datacenter',
+         'PDS operator side': 'On-prem centralized',
+         'on-prem': 'On-prem centralized',
+         'on-prem-only': 'On-prem centralized',
+         'azure': 'Microsoft Azure',
+         'aws': 'Amazon Web Services',
+         'plesk': 'Plesk',
+         'ios': 'iOS Devices',
+         'android': 'Android Devices',
+         "vmware appliance": "VMWare Appliance",
+         'Agent': 'On-prem agent unit',
+         'TBD': 'TBD'
+         }
+
+    langs = {'Go-lang', 'Python', 'C/C++', 'Javascript', 'Java', 'Ruby', 'Erlang', 'shellscript', 'Perl',
+              'Objective-C', 'Android', 'Powershell-script', '.Net', 'PHP'}
+
+    app_vendors = {'Acronis', 'OpenSource', 'Percona', 'Google', 'Commerical'}
+
+    app_frameworks = \
+        {'tornado': 'Python', 'sqlalchmey': 'Python', 'fastcgi': 'Python', 'storedprocs': 'Python',
+         'aiohttp': 'Python', 'angular4': 'Javascript', 'vuejs': 'Javascript', 'angular1': 'Javascript',
+         'laravel': 'PHP', 'werkzeug': 'Python', 'flask': 'Python', 'go-std': 'Go-lang',
+         'beego': 'Go-lang', 'xorm': 'Go-lang', 'gorm': 'Go-lang', 'WPF': '.Net', 'angular2': 'Javascript',
+         'extjs4': 'Javascript', 'gevent': 'Python', 'ruby on rails': 'Ruby', 'android sdk': 'Android',
+         'wix': 'C/C++', 'phantomjs': 'Python', 'asyncio': 'Python', 'go-micro': 'Go-lang',
+         'gin': 'Go-lang', 'gorp': 'Go-lang', 'DBIx': 'Perl', 'Mason': 'Perl', 'boost': 'C/C++'}
+
+    prod_versions = {
+        'Company Cloud 1.0': {'shortname': 'ComCloud1.0', 'order': 1, 'family': 'Cloud Services'},
+        'Google Maps 1.0': {'shortname': 'Maps1.0', 'order': 2, 'family': 'Cloud Services'}}
+
+    prod_families = {'Cloud Services', 'Native Applications'}
+
+    ports = \
+        {'HTTP-80': {'port': 80}, 'HTTPS-443': {'port': 443}, 'HTTP-8080': {'port': 8080}}
+
+    # TBD: How to make sure these ones stay valid with respective seed values.
+    default_depl_env = 'K8S pod'
+    default_location = 'Cloud datacenter'
+    default_prod_version = 'Company Cloud 1.0'
+    default_prod_family = 'Cloud Services'
+    default_port = 'HTTPS-443'
+    default_runtime = 'Component'
+    default_app_vendor = 'OpenSource'
+    default_dep_type = 'sync_rw'
+    default_privacy = 'Application'
+    default_comp_version = '1.0'
+
+    @staticmethod
+    def usage():
+        print(r"Usage: ", file=sys.stderr)
+        print (r"Application to import CSV data into DB.")
+        print(r"# python ./manage.py seed-importer.py /path/to/seeddata.csv /path/to/defaults.json", file=sys.stderr)

```

Comment:
Why we need both *.csv and *.json files?

Reply:
I began with CSV file which cannot have multiple sheets to represent seed data for all tables effectively. Hence came up with defaults.json.

It will converge to one xlsx file in the next version with different component details format. Till then we'll have 2 files with input data.